app/_src/gateway/production/monitoring/health-check.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Health Check
+content_type: tutorial
+---
+
+This tutorial will guide you through the process of using the Node Readiness Endpoint, which provides a reliable and simple way to determine if Kong is ready to serve user requests.
+
+The health check endpoint returns a `200 OK` response when Kong is ready, or a `503 Service Temporarily Unavailable` response when it's not. This is useful for load balancers and other tools that need to monitor the readiness of Kong instances.
+
+Note that the readiness endpoint does not return detailed node health information such as the
+amount of memory Kong node is using. You might want to check out the [`/status` API](/gateway/{{page.kong_version}}/admin-api/#retrieve-node-status) as well.
+
+## Prerequisites
+
+* Kong installed
+* Basic understanding of Kong configuration and deployment modes (Traditional, DB-less, and Hybrid)
+
+## Understanding the Node Readiness Endpoint
+
+Before diving into the steps, it's important to understand the purpose of the Node Readiness Endpoint and how it determines whether a Kong instance is ready or not.
+
+### Traditional mode
+
+In Traditional mode, the endpoint returns `200 OK` when all of the following conditions are met:
+
+1. Successful connection to the database
+2. Successful initial build of all routers for all workers
+3. Successful initial build of all plugin iterators for all workers
+
+### DB-less mode (`data_plane` role)
+
+In DB-less mode, the endpoint returns `200 OK` when all of the following conditions are met:
+
+1. Kong has loaded a valid and non-empty config
+2. Successful initial build of all routers for all workers
+3. Successful initial build of all plugin iterators for all workers
+
+## Enabling the Status Endpoint
+
+In order to use the Node Readiness Endpoint, make sure that you have enabled the Status API Server via the [status_listen](/gateway/{{page.kong_version}}/reference/configuration/#status_listen) configuration parameter.
+
+Example `kong.conf`:
+
+```conf
+status_listen = 0.0.0.0:8100
+```
+
+## Using the Node Readiness Endpoint
+
+Once you've enabled the Node Readiness Endpoint, you can send a GET request to it to check the readiness of your Kong instance:
+
+```sh
+# Replace `<status-api-host>` with the appropriate host for
+# your Status API server, including the port number
+
+curl -i <status-api-host>/status/ready
```

Comment:
I think we are missing a new page for the entire Status API as added by https://github.com/Kong/kong/pull/4977 - and of which `/status/ready` is now a part of. Currently, `/status` is only documented in https://docs.konghq.com/gateway/latest/reference/configuration/#status_listen. A more accurate representation would be to have `/status/` and `/status/ready` in a new overall page separate from "Admin API" too. If the user configures `status_listen`, there will be a separate port for status checking that includes both `/status/` and `/status/ready`.